src/Dependency/InheritanceFlatter.php

```diff
@@ -16,6 +16,7 @@ public function flattenDependencies(
             foreach ($astMap->getClassInherits($classReference->getClassName()) as $inherit) {
                 foreach ($dependencyResult->getDependenciesByClass($inherit->getClassName()) as $dep) {
                     $dependencyResult->addInheritDependency(new InheritDependency(
+                        $classReference->getClassName() . '.php',
```

Comment:
`InheritDependency` does not need a path as the original dependency/violation reference should be used instead